sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenter.java

```diff
@@ -40,15 +41,17 @@
   private Version installedSonarVersion;
   private Date date;
   private Sonar sonar;
+  private List<Scanner> scanners;
 
-  private UpdateCenter(PluginReferential updateCenterPluginReferential, Sonar sonar) {
+  private UpdateCenter(PluginReferential updateCenterPluginReferential, List<Scanner> scanners, Sonar sonar) {
     this.updateCenterPluginReferential = updateCenterPluginReferential;
     this.sonar = sonar;
     this.installedPluginReferential = PluginReferential.createEmpty();
+    this.scanners = scanners;
   }
 
-  public static UpdateCenter create(PluginReferential updateCenterPluginReferential, Sonar sonar) {
-    return new UpdateCenter(updateCenterPluginReferential, sonar);
+  public static UpdateCenter create(PluginReferential updateCenterPluginReferential, List<Scanner> scanners, Sonar sonar) {
+    return new UpdateCenter(updateCenterPluginReferential, scanners, sonar);
```

Comment:
This is a breaking change. This would imply we increment the major version of this release. I'm fine with it, but I would check with a SQ backend dev to have their opinion on it.

Reply:
I did a search on SQ and found no usage of this method but will do an extra check.

Reply:
Even if it's not, it's still a public API, which now has changed 🤷‍♂ . Better to increment the major version.

Reply:
Yes sorry I am for it for the new version